markets/perps-market/contracts/interfaces/IMarketConfigurationModule.sol

```diff
@@ -91,15 +193,37 @@ interface IMarketConfigurationModule {
             uint256 maxSecondsInLiquidationWindow
         );
 
+    /**
+     * @notice Gets funding parameters of a market.
+     * @param marketId id of the market.
+     * @return skewScale the skew scale.
+     * @return maxFundingVelocity the max funding velocity.
+     */
     function getFundingParameters(
         uint128 marketId
     ) external view returns (uint256 skewScale, uint256 maxFundingVelocity);
 
+    /**
+     * @notice Gets the max market value of a market.
+     * @param marketId id of the market.
+     * @return maxMarketValue the max market value.
+     */
     function getMaxMarketValue(uint128 marketId) external view returns (uint256 maxMarketValue);
```

Comment:
Maybe mention: 
Max Market Value controls the maximum open interest a market an have on either side (Long | Short). So the total Open Interest for a market is Max Market Value * 2. This value is not defined in USD but in the native asset